todo-app/src/storage.js

```diff
@@ -0,0 +1,8 @@
+export function loadTodoItems() {
+   if(localStorage.todos == null){ return [] }
```

Comment:
Careful here. You should try to use strict equality checks wherever possible. In fact, `localStorage.tasks` will never be `null` unless you set it to `null`, it will be `undefined` by default. The only reason this loose comparison works is because `undefined` and `null` are both falsey.

You can check out the [MDN docs here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) for some good ideas about how to check against `undefined` in JavaScript.